### PR TITLE
Fixed command bugs.

### LINKED ISSUE #173: [PE-D] Editing a previously empty field will add it
Initially, I didn't input a work address for zima (can refer to previous screenshots). When editing that field it adds the work address. This does not seem to be a normal functionality of an `edit` method. If yall intent to do it this way maybe can consider putting this behaviour in UG? so that it can become a feature.

On a side note, the message displayed is not very specific. A better way is to just display the edited field of that person.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: charliemoweng/ped#8

### LINKED ISSUE #176: [PE-D] fcall after scall still update the failed attempt
Using fcall after the person has been marked as successfully called still results in the person being fcall-ed unsuccessfully. 

My opinion is that after the person has been successfully called, there should not be further failed communication as this could discriminate against those that have been successfully scall-ed already.

The argument could be that after calling successfully, the person could not be contacted further.

-------------
Labels: `severity.VeryLow` `type.FeatureFlaw`
original: kanjitp/ped#7

### LINKED ISSUE #183: [PE-D] Delete Command with multiple indexes specified not function as expected
From the UserGuide, delete 1 5 4 would delete the 1st, 4th, and 5th person. 
And upon experimenting with the software, delete 1 1 1 would delete the first three people.
However, when keying in `delete 2 1 1 2 2 1`, the deleted people are not the 1st, 2nd, 3rd, 5th, 6th, and 6th as expected (I assume it should be the same as `delete 1 1 1 2 2 2`, so would be [1, 2, 3, 5, 6, 7]). The executing sequence of delete multiple people is kind of confusing.

Below is a record for command `2 1 2 1 1 2` (expect the 4th person to remain)

https://user-images.githubusercontent.com/46596402/139413285-6566d817-6daf-41b0-8b66-7558c0c17f33.mov

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: NoraYUuu/ped#6

### PR DESCRIPTION
**Pull request by luffingluffy**
Fixes #183.
Fixes #176.
Fixes #173.

### CODE REVIEW

File: src/main/java/seedu/track2gather/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -63,7 +63,7 @@
             + PREFIX_PHONE + "91234567 "
             + PREFIX_EMAIL + "johndoe@example.com";
 
-    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s\n"
+    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Fields%1$s\n"
```
Comment (jennibearduit) [inline src/main/java/seedu/track2gather/logic/commands/EditCommand.java:66]: Missing space here?

**Review by jennibearduit — APPROVED** [review]
Minor space issue, but overall LGTM!  Feel free to merge after you resolve it

File: src/main/java/seedu/track2gather/logic/commands/EditCommand.java (hunk 1 above)
Comment (luffingluffy) [reply src/main/java/seedu/track2gather/logic/commands/EditCommand.java:66]: It is intended since my string builder adds in the space.

Comment (jennibearduit) [reply src/main/java/seedu/track2gather/logic/commands/EditCommand.java:66]: Ah okie!

**Pull request merged**